<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Football League</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 255, 76);
alternate-background-color: rgb(208, 208, 208);
border-right-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome To The Champions League!</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="SimulateButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 172, 5);</string>
      </property>
      <property name="text">
       <string>Simulate Rounds</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SearchButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 172, 5);</string>
      </property>
      <property name="text">
       <string>Search for a team</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 172, 5);</string>
      </property>
      <property name="text">
       <string>Reset Stats</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="GoalsScorersLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>525</width>
     <height>265</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>581</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Team</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pl</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>W</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>L</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>GF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>GA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pts</string>
    </property>
   </column>
  </widget>
  <widget class="QTextBrowser" name="RoundResultLog">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>525</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Top 5 Goal Scorers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rounds Results</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>326</y>
     <width>825</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>10</y>
     <width>20</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
